project/plugins.sbt

```diff
@@ -1,11 +1,17 @@
+enablePlugins(BuildInfoPlugin)
+
+val sbtProtocV = "0.99.26"
+
+buildInfoKeys := Seq[BuildInfoKey]("sbtProtocVersion" -> sbtProtocV)
+
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.3.0")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.4.0")
-addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.19")
+addSbtPlugin("com.thesamet" % "sbt-protoc" % sbtProtocV)
```

Comment:
Weird to have one version that is only used once in a val but not the others?

Reply:
It's used in the `buildInfoKeys` as well, which makes it available to the main project build (#765)

Reply:
Ah, didn't connect those dots